apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,111 +18,174 @@ export function InstructorCourseAdminSettings({
       </head>
       <body>
         ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
-        <main id="content" class="container-fluid">
+        <main id="content" class="container mb-4">
           ${renderEjs(__filename, "<%- include('../partials/courseSyncErrorsAndWarnings'); %>", {
             ...resLocals,
           })}
-          <div class="card mb-4">
-            <div class="card-header bg-primary text-white d-flex">Settings</div>
-            <table class="table table-sm table-hover two-column-description">
-              <tbody>
-                <tr>
-                  <th>Short Name</th>
-                  <td>${resLocals.course.short_name}</td>
-                </tr>
-                <tr>
-                  <th>Title</th>
-                  <td>${resLocals.course.title}</td>
-                </tr>
-                <tr>
-                  <th>Institution</th>
-                  <td>${resLocals.institution.short_name} (${resLocals.institution.long_name})</td>
-                </tr>
-                <tr>
-                  <th>Timezone</th>
-                  <td>${resLocals.course.display_timezone}</td>
-                </tr>
-                <tr>
-                  <th>Path</th>
-                  <td>${resLocals.course.path}</td>
-                </tr>
-                <tr>
-                  <th>Repository</th>
-                  <td>${resLocals.course.repository}</td>
-                </tr>
-                <tr>
-                  <th>Configuration</th>
-                  <td>
-                    ${!coursePathExists
+          <div class="card">
+            <div class="card-header bg-primary text-white d-flex">Course Settings</div>
+            <div class="card-body">
```

Comment:
You're missing a corresponding closing `</div>` tag after `</form>`.

Reply:
Good catch. Added in [ad5d7a8](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/ad5d7a8bae4c6acd3db8daddaa0e356b0b47ee2b).